src/content_mapblock.cpp

```diff
@@ -559,7 +543,7 @@ void MapblockMeshGenerator::drawLiquidSides(bool flowing)
 	}
 }
 
-void MapblockMeshGenerator::drawLiquidTop(bool flowing)
+void MapblockMeshGenerator::drawLiquidTop()
```

Comment:
Could you please rename these functions to `drawFlowingLiquid{*}`, for easier understanding?